src/common/components/game/GameArea.tsx

```diff
@@ -279,6 +280,7 @@ export default class GameArea extends React.Component<GameAreaProps, GameAreaSta
             winnerName={(winner && winner !== 'draw') ? usernames[winner] : null}
             onClick={onClickEndGame}
           />
+          {tutorialStep && tutorialStep.idx === 0 && <TutorialIntroScreen onClickEndGame={onClickEndGame} />}
```

Comment:
Ah, so we're not using `tutorialScript[0]` in `tutorial.ts` at all now? Maybe you should just get rid of the first tutorial step completely or make it clear that it's unused now.

Reply:
Oh we still are. This was just my way of checking that we're at the beginning of the tutorial